tests/v4/functional/test_redact_folder.py

```diff
@@ -63,10 +65,9 @@ def _replace_file_ext(file_path: Union[str, Path], new_ext: str = ".json") -> st
         "output_type,service,file_extension",
         [
             [OutputType.labels, ServiceType.redact_area, ".json"],
-            [OutputType.overlays, ServiceType.dnat, ".jpg"],
+            [OutputType.overlays, ServiceType.dnat, ".apng"],
```

Comment:
this test fails locally, 
`redact/v4/tools/redact_file._get_out_path` returns
`/tmp/pytest-of-ra/pytest-7/test_image_correct_file_ending0/obama_redacted.png`
instead of `apng`.

Could you please check the test locally as well?
Is it my environment of settings, or is it an actual issue?